README.md

```diff
@@ -27,6 +27,14 @@ To use your own config file, set the `LOGSTASH_CONFIG_URL` environment variable
       -p 9200:9200 \
       pblittle/docker-logstash
 
+To use a config file from the local file system, mount the file as a volume using the `-v` option. Replace `/my/logstash.conf` with the absolute path to your config file.
```

Comment:
@remmelt, to keep the documentation uniform, please use the `<your_whatever>` syntax. Something like `<your_logstash_config>`.